tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -2439,20 +2439,20 @@ arguments when you emit signals; it's up to you to emit the correct values.
 
 .. image:: img/gdscript_basics_signals_node_tab_1.png
 
-GDScript can bind an array of values to connections between a signal
-and a method. When the signal is emitted, the callback method receives
-the bound values. These bound arguments are unique to each connection,
-and the values will stay the same.
+You can also create copies of GDScript Callable objects which accept additional
+arguments using :ref:`Callable.bind() <class_Callable_method_bind>`. This
+allows you to add extra information to the connection if the emitted signal
+itself doesn't give you access to all the data that you need.
 
-You can use this array of values to add extra constant information to the
-connection if the emitted signal itself doesn't give you access to all the data
-that you need.
+When the signal is emitted, the callback method receives the bound values, in
+addition to those provided by the signal. These bound arguments are unique to
+each connection, and the values will stay the same.
```

Comment:
This might be confusing for data that is shared, like resources, which are shared even in binds, depending on the context, not sure if `Array`s are shared in this case or made unique

Reply:
Verified the behavior with resources and `Array`s, both are shared, not made unique and affected by side effects in the callbacks. Removed the sentence.